### PR TITLE
Updating Unicode.md

### LINKED ISSUE #966: Wrong link for DejaVu fonts in docs
https://github.com/py-pdf/fpdf2/blob/a4d05f5f5cdfd6451441f9465e1ae65d438836f8/docs/Unicode.md?plain=1#L15

I believe the correct link should be: `https://dejavu-fonts.github.io/`

The current link goes to a french blog. Kinda looks like someone hijacked the domain, but I dunno.

### PR DESCRIPTION
**Pull request by gmischler**
e.g. Fixes #966

**Checklist**:

- [x] This PR is ready to be merged 

This was sorely needed.

* Some information at the top of the file seemed to have been an unchanged copy of the original PHP version, and completely meaningless to our users.
* I've added references to text shaping where relevant.
* I think nowadays it makes more sense to link to some largish font collections than to individual fonts.
* I'm not sure if the RTL section is still current with text shaping available (it will become entirely irrelevant with #972).

By submitting this pull request, I confirm that my contribution is made under the terms of the [GNU LGPL 3.0 license](https://github.com/py-pdf/fpdf2/blob/master/LICENSE).

### CODE REVIEW

File: docs/Unicode.md [hunk 1]
```diff
@@ -1,47 +1,81 @@
...
-(ttf-indic-fonts Debian and Ubuntu package) for Bengali, Devanagari, Gujarati,
-Gurmukhi (including the variants for Punjabi), Kannada, Malayalam, Oriya,
-Tamil, Telugu, Tibetan
+To use a Unicode font in your program, use the [`add_font()`](fpdf/fpdf.html#fpdf.fpdf.FPDF.add_font), and then the  [`set_font()`](fpdf/fpdf.html#fpdf.fpdf.FPDF.set_font) method calls.
+
+
+### Built-in Fonts vs. Unicode Fonts ###
+
+The PDF file format knows a small number of "standard" fonts, namely "courier", "helvetica", "times", "symbol", and "zapfdingbats". The first three are available in regular, bold, italic, and bold-italic versions. Any PDF processor (eg. a viewer) must provide those fonts for display. To use them, you don't need to call `.add_font()`, but only `.set_font()`.
+
```
Comment (andersonhc) [inline docs/Unicode.md:25]: Do you think it's work adding:

On fpdf, "arial" is an alias for the standard font "helvetica".

File: docs/Unicode.md [hunk 2]
```diff
@@ -52,7 +86,9 @@ From the error in the image above, Thai characters can be fixed using fonts from
 
 ![thai-font-working](https://raw.githubusercontent.com/py-pdf/fpdf2/master/tutorial/thai-accent-working.png)
```
Comment (andersonhc) [inline docs/Unicode.md:88]: This Thai font problem doesn't happen when text shaping is enabled

File: docs/Unicode.md [hunk 3]
```diff
@@ -140,17 +176,11 @@ pdf.output("unicode.pdf")
 View the result here: 
 [unicode.pdf](https://github.com/py-pdf/fpdf2/raw/master/tutorial/unicode.pdf)
```
Comment (andersonhc) [inline docs/Unicode.md:178]: If all the text if right-to-left and text shaping is enabled there is no need for this workaround. However if there is any bidirectional segment (RTL inside a LTR text or the other way around) fpdf will render in a single direction for now.

**Review by andersonhc — APPROVED** [review]

File: docs/Unicode.md (hunk 1 above)
Comment (gmischler) [reply docs/Unicode.md:25]: Hmmm.... Didn't we want to remove those aliases?

File: docs/Unicode.md (hunk 2 above)
Comment (gmischler) [reply docs/Unicode.md:88]: So most of this section (except the first paragraph) can go?

File: docs/Unicode.md (hunk 3 above)
Comment (gmischler) [reply docs/Unicode.md:178]: So the workaround is only needed for mixed direction text?

We also currently still have #901, which we should probably mention in this context.

File: docs/Unicode.md (hunk 1 above)
Comment (andersonhc) [reply docs/Unicode.md:25]: We can probably deprecate, removing it would break programs for many of our users.

File: docs/Unicode.md (hunk 2 above)
Comment (andersonhc) [reply docs/Unicode.md:88]: Correct

File: docs/Unicode.md (hunk 3 above)
Comment (andersonhc) [reply docs/Unicode.md:178]: #901 is also being fixed on the bidirectional PR.
I guess we can leave this section as-is and I change this document on my PR.

File: docs/Unicode.md (hunk 1 above)
Comment (gmischler) [reply docs/Unicode.md:25]: Given that it was never documented, I'm not really sure, but that's a topic for a different PR.

**Pull request merged**